sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RedundantNullCheck.Fixed.Batch.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Tests.Diagnostics
+{
+    public class RedundantNullCheck
```

Comment:
silly question from me: what's the difference between Batch and non-Batch?

Reply:
Files are the same. From what I udnerstand, it's the way the code fix provider is executed that is differrent: depending whether the user wants to fix all the issues at the same time or not.
But resulting fixed file here are indeed identical